packages/openneuro-app/src/scripts/dataset/comments/coral-embed.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useRef, useState } from "react"
+
+declare global {
+  interface Window {
+    Coral?: {
+      createStreamEmbed: (config: any) => void
+    }
+  }
+}
+
+export const CoralEmbed: React.FC<{ storyID: string }> = ({ storyID }) => {
+  const coralContainerRef = useRef<HTMLDivElement>(null)
+  const [coralSSOToken, setCoralSSOToken] = useState<string | null>(null)
+  const isAuthenticated = document.cookie.includes("accessToken=")
+
+  useEffect(() => {
+    const fetchAndInitializeCoral = async () => {
+      const accessTokenFromCookie = document.cookie.split("; ").find((cookie) =>
+        cookie.startsWith("accessToken=")
+      )?.split("=")[1]
+      const headers = accessTokenFromCookie
+        ? { Authorization: `Bearer ${accessTokenFromCookie}` }
+        : {}
```

Comment:
This shouldn't be necessary. The cookie is already set when the fetch request is made.

Reply:
I think I have made the right adjustments.